main.js

```diff
@@ -58,11 +58,39 @@ BlogPost.prototype.showUserAuthorsName = function(parent) {
   parent.appendChild(showAuthorName);
 };
 
+BlogPost.prototype.showUserTagFilter = function(parent, tag) {
+  const showUserTagName = document.createElement("div");
+  showUserTagName.dataset.tag = this.tags;
+  showUserTagName.id = "tag-name";
```

Comment:
what purpose does this id and class name serve? does this element need these things?